### PR TITLE
Move Cobs Wire Worker behind feature flag.

### PR DESCRIPTION
**Pull request by royb3**
Hi, I tried to use Postcard-RPC in a project. I think it's actually pretty nice to use, but I have no need for the serial COBS worker or the Tokio serial dependency. I moved the Tokio serial stuff to a new mod and put that behind a feature flag.

Do you think this is a good idea, or would you solve this in a different way?

P.S. I did remove the deprecated new constructor.

### CODE REVIEW

File: source/postcard-rpc/src/lib.rs [hunk 1]
```diff
@@ -175,7 +175,9 @@ use headered::extract_header_from_bytes;
 use postcard::experimental::schema::Schema;
 use serde::{Deserialize, Serialize};
 
+#[cfg(feature = "cobs")]
```
Comment (jamesmunns) [inline source/postcard-rpc/src/lib.rs:178]: Should this be `cobs-serial`?

**Review by jamesmunns — COMMENTED** [review]
Looks fine to me, thanks! I'd like to see a mention of the feature in either the `lib.rs` doc comments or the README so this is clear.

Once that and the other comment is resolved I can merge and publish.

File: source/postcard-rpc/src/lib.rs (hunk 1 above)
Comment (royb3) [reply source/postcard-rpc/src/lib.rs:178]: Yup, it is indeed.

**Comment by jamesmunns** [comment]
Thank you!

**Pull request merged**